main.lua

```diff
@@ -1,4 +1,4 @@
-VERSION = "0.4.3"
+VERSION = "0.4.4"
```

Comment:
I would undo this, since I wanted to release this together with the references feature, making it a 0.5.0